notebooks/adhoc/jovian_analysis/jovian_src/analysis_functions.py

```diff
@@ -25,7 +24,8 @@
 class JovianAnalysisResult:
     """Results from analyzing blocks with a specific DA footprint gas scalar."""
     da_footprint_gas_scalar: int
-    gas_limit: int
+    gas_limit: int  # Most common gas limit from block analyses
```

Comment:
To calculate the "effective limit" we should use the latest, not the most common, gas_limit.